src/CoreEditor/webrunesAPI.js

```diff
@@ -35,8 +35,8 @@ export function getWidgetID(url: string): Promise {
     .withCredentials();
 }
 
-// export function getRegistredUser(): Promise {
-//   return request
-//     .get(`${protocol()}//login.${domain()}/api/get_profile`)
-//     .withCredentials();
-// }
+export function getRegistredUser(): Promise {
```

Comment:
We have no working login.wrioos.com so it should be commented off. Just left the login button itself for later usage.